projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.ts

```diff
@@ -359,15 +357,15 @@ export class IgxDateTimeEditorDirective extends IgxMaskDirective implements OnCh
       return { value: true };
     }
 
-    const maxValueAsDate = this.isDate(this.maxValue) ? this.maxValue : this.parseDate(this.maxValue);
-    const minValueAsDate = this.isDate(this.minValue) ? this.minValue : this.parseDate(this.minValue);
+    const maxValueAsDate = isDate(this.maxValue) ? this.maxValue : this.parseDate(this.maxValue);
```

Comment:
I realize this hasn't changed, but there's an edge case w/ invalid date that'd fail here, since both options aren't equipped to handle it.